packages/ionic/src/commands/cordova/prepare.ts

```diff
@@ -37,6 +38,7 @@ You may wish to use ${chalk.green('ionic cordova prepare')} if you run your proj
           default: true,
         },
         ...APP_SCRIPTS_OPTIONS,
+        ...ANGULAR_BUILD_COMMAND_OPTIONS,
```

Comment:
This also shouldn't be here, exactly, but it's my fault. I must've missed the `APP_SCRIPTS_OPTIONS` for cordova prepare. So, it's fine for now.